### ISSUE TITLE
nested comments bug

### ISSUE THREAD

**Issue by t4wE** [vlang/v#14695]
**v doctor**
OS: windows,  Windows 10 Pro v19043 64-bit
Processor: 4 cpus, 64bit, little endian, Intel(R) Core(TM) i3-4130 CPU @ 3.40GHz
CC version: Error: exec failed (CreateProcess) with code 2: The system cannot find the file specified.
 cmd: cc --version

getwd: C:\Users\t4wE\Desktop\v
vmodules: C:\Users\t4wE\.vmodules
vroot: C:\Users\t4wE\Desktop\v
vexe: C:\Users\t4wE\Desktop\v\v.exe
vexe mtime: 2022-06-06 07:12:22
is vroot writable: true
is vmodules writable: true
V full version: V 0.2.4 82eb495.56f3817

Git version: git version 2.35.1.windows.2
Git vroot status: weekly.2022.22-62-g56f38175
.git/config present: true
thirdparty/tcc status: thirdparty-windows-amd64 125e21e2

**What did you do?**
/*
comment1
/*
comment2
*/

fn main() {
  println('some output')
}

**What did you expect to see?**
some output
 
**What did you see instead?**
no output

**Comment by JalonSolov** [comment]
The only output should be an `unterminated multi-line comment` error.

**Comment by Larpon** [comment]
No, the program should compile and run since V supports nested comments and the second `/*` is in a block comment IMO

**Comment by Larpon** [comment]
Everything between `/*` and `*/` is insignificant and should be ignored

**Comment by JalonSolov** [comment]
There's a bug either way.  :-)

**Comment by felipensp** [comment]
This happens because V implements nested comment, i.e. it makes working:

```
/*
comment1
/*
comment2
*/
*/

fn main() {
  println('some output')
}
```
